### ISSUE TITLE
i cant open files

### ISSUE THREAD

**Issue by Albert7758** [Ultimaker/Cura#1467]

**Comment by fieldOfView** [comment]
We need more details. What version of Cura on what OS? How do you try to open files? Got logs?

**Comment by Ghostkeeper** [comment]
Can't reproduce this, files open fine. We'll need more specifics.

**Comment by fieldOfView** [comment]
I *think* this might be the bug where on the first launch on Windows, you cannot drag and drop files into Cura. This is happens only once (when running Cura from the installer) and is automagically solved by restarting Cura.

That, or @Albert7758 has glued his folders shut with the glue-stick that came with his printer and he cannot access the files inside them. I can't tell.

**Comment by Ghostkeeper** [comment]
Just the other day I found an old binder with some files under my bed, with some physics exercises from high school. The binder with files opened fine but it was a little dusty. If he used the glue stick that might've been a problem.

**Comment by fieldOfView** [comment]
@Albert7758, we're not making fun of you (well, ok, a little). Please let us know if my first suggestion was indeed what you meant, or if you have found another issue.
